### PR TITLE
Select2 update

### PR DESCRIPTION
**Pull request by NejcZdovc**
Updated select2 to version 0.4.1.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,7 +51,6 @@
     "zone.js": "^0.6.12"
   },
   "scripts": {
-    "postinstall": "typings install",
```
Comment (fjozsef) [inline package.json]: I think, we stil need this for this commit.

Comment (NejcZdovc) [inline package.json]: I was still in the middle of rebase, you were too fast. It should be fine now.

Comment (fjozsef) [inline package.json]: You still need to rebase

**Comment by NejcZdovc** [comment]
Please see PR #77. Closing this PR.

**Pull request closed without merging**